tests/run.rs

```diff
@@ -479,6 +479,26 @@ fn lib_with_path_dep_sdist() {
     ))
 }
 
+#[test]
+fn lib_with_target_path_dep() {
+    handle_result(other::test_source_distribution(
+        "test-crates/sdist_with_target_path_dep",
+        vec![
+            "sdist_with_target_path_dep-0.1.0/local_dependencies/some_path_dep/Cargo.toml",
+            "sdist_with_target_path_dep-0.1.0/local_dependencies/some_path_dep/src/lib.rs",
+            "sdist_with_target_path_dep-0.1.0/local_dependencies/transitive_path_dep/Cargo.toml",
+            "sdist_with_target_path_dep-0.1.0/local_dependencies/transitive_path_dep/src/lib.rs",
+            "sdist_with_target_path_dep-0.1.0/Cargo.toml",
+            "sdist_with_target_path_dep-0.1.0/Cargo.lock",
+            "sdist_with_target_path_dep-0.1.0/pyproject.toml",
+            "sdist_with_target_path_dep-0.1.0/src/lib.rs",
+            "sdist_with_target_path_dep-0.1.0/PKG-INFO",
+        ],
+        None,
```

Comment:
Can you add an assertion for rewritten `Cargo.toml` content?

Reply:
Of course: https://github.com/PyO3/maturin/blob/110625e66d81bf2cf23d39b065a1faf2e5a1730a/tests/run.rs#L486-L500